python/setup.py

```diff
@@ -201,6 +201,15 @@ def get_package_data_files():
     return ["relay/std/prelude.rly", "relay/std/core.rly"]
 
 
+def long_description_contents():
+    with open(
+        os.path.abspath(os.path.join(CURRENT_DIR, "..", "README.md")), encoding="utf-8"
```

Comment:
nit: `pathlib` makes life easier 

```suggestion
        pathlib.Path(CURRENT_DIR).resolve().parent / "README.md", encoding="utf-8"
```